crates/redeem/src/types.rs

```diff
@@ -50,12 +50,14 @@ pub struct RedeemRequest<AccountId, BlockNumber, PolkaBTC, DOT> {
     #[cfg_attr(feature = "std", serde(deserialize_with = "deserialize_from_string"))]
     #[cfg_attr(feature = "std", serde(bound(serialize = "PolkaBTC: std::fmt::Display")))]
     #[cfg_attr(feature = "std", serde(serialize_with = "serialize_as_string"))]
-    /// Total redeem fees in PolkaBTC - taken from request amount
+    /// Total parachain fee in PolkaBTC - taken from request amount
     pub fee: PolkaBTC,
     #[cfg_attr(feature = "std", serde(bound(deserialize = "PolkaBTC: std::str::FromStr")))]
     #[cfg_attr(feature = "std", serde(deserialize_with = "deserialize_from_string"))]
     #[cfg_attr(feature = "std", serde(bound(serialize = "PolkaBTC: std::fmt::Display")))]
     #[cfg_attr(feature = "std", serde(serialize_with = "serialize_as_string"))]
+    /// Amount the vault should spend on the bitcoin inclusion fee - taken from request amount
+    pub transfer_fee_btc: PolkaBTC,
```

Comment:
We need to do a runtime upgrade.

Reply:
I'm not sure I follow - there already is a runtime upgrade function defined that sets the `transfer_fee_btc` to 0 for existing requests

Reply:
Ah I missed that, okay ignore this :)